web/styles/settings.css

```diff
@@ -1,7 +1,7 @@
 /* The height of the settings header (including tab switcher). */
 $settings_header_height: 3.2142em; /* 45px / 14px em */
 /* The width of the settings sidebar. */
-$settings_sidebar_width: 255px;
+$settings_sidebar_width: 18.2142em; /* 255px / 14px em */
```

Comment:
Is this the right away to go? Or would it be better to express as a percentage?

Reply:
Okay. Also, could you note your px-to-em conversions like so: `255px at 14px/1em`?

Reply:
Oh yes, sorry I probably opened this PR before I had a convention for this.